modules/efk-logging-configuring-image-about.adoc

```diff
@@ -25,23 +25,11 @@ OAUTH_PROXY_IMAGE=registry.redhat.io/openshift4/ose-oauth-proxy:v4.1 <5>
 <3> *KIBANA_IMAGE* deploys Kibana.
 <4> *CURATOR_IMAGE* deploys Curator.
 <5> *OAUTH_PROXY_IMAGE* defines OAUTH for OpenShift Container Platform.
-
-[NOTE]
-====
-The values might be different depending on your environment.
-====
-
-
-
-////
-Comment out until 4.1
-* *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`. <1>
-
-<1> The image used for RSYSLOG when deployed. You can change this value using an environment variable. You cannot change this value through the Cluster Logging CR.
+<6> *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`. You can change this value using an environment variable. You cannot change this value through the Cluster Logging CR.
```

Comment:
I don't think the default value of RSYSLOG_IMAGE will be `docker.io/viaq/rsyslog:latest` after the rsyslog's GA. And the RSYSLOG_IMAGE is written in the CSV, it's better not to change the value. Setting environment variable won't make any change.